src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -60,13 +60,15 @@ function ProviderSelectionField({
 
   useEffect(
     () => {
-      if (sortMethod === FACILITY_SORT_METHODS.distanceFromCurrentLocation) {
-        requestProvidersList(currentLocation);
-      } else {
-        requestProvidersList(address);
+      if (showProvidersList) {
+        requestProvidersList(
+          sortMethod === FACILITY_SORT_METHODS.distanceFromCurrentLocation
+            ? currentLocation
+            : address,
+        );
       }
     },
-    [sortMethod],
+    [sortMethod, showProvidersList],
```

Comment:
Providers were being fetched on page load before the user clicks "Choose a provider". This change checks if `showProvidersList` is true before fetching.

@vbahinwillit was this intentional? Not sure if we wanted to pre-fetch this data or not.

Reply:
That was intentional, though now that the PPMS call is faster, it's maybe not as necessary.